### ISSUE TITLE
Flywheel should not be required in a dedicated server

### ISSUE THREAD

**Issue by ccrvincent** [talrey/CreateDeco#69]
Flywheel has specifically rejected bugs when running on a server, so CreateDeco should not rely on its presence in a dedicated server environment.

```
[modloading-worker-0/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to create mod instance. ModID: createdeco, class com.github.talrey.createdeco.CreateDecoMod
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.2.2.jar%23103!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.2.2.jar%23102!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.NoClassDefFoundError: com/jozufozu/flywheel/core/StitchedSprite
	at com.simibubi.create.foundation.block.render.SpriteShiftEntry.set(SpriteShiftEntry.java:13) ~[create-1.18.2-0.5.0.i.jar%2347!/:0.5.0.i]
	at com.simibubi.create.foundation.block.connected.CTSpriteShifter.getCT(CTSpriteShifter.java:20) ~[create-1.18.2-0.5.0.i.jar%2347!/:0.5.0.i]
	at com.github.talrey.createdeco.connected.SpriteShifts.populateMaps(SpriteShifts.java:27) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.connected.SpriteShifts.<clinit>(SpriteShifts.java:19) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.registry.SheetMetal.buildSheetMetalBlock(SheetMetal.java:74) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.registry.SheetMetal.lambda$registerBlocks$31(SheetMetal.java:209) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
	at com.github.talrey.createdeco.registry.SheetMetal.registerBlocks(SheetMetal.java:204) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.Registration.registerBlocks(Registration.java:396) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.CreateDecoMod.<init>(CreateDecoMod.java:52) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	... 14 more
Caused by: java.lang.ClassNotFoundException: com.jozufozu.flywheel.core.StitchedSprite
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:134) ~[securejarhandler-1.0.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:134) ~[securejarhandler-1.0.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at com.simibubi.create.foundation.block.render.SpriteShiftEntry.set(SpriteShiftEntry.java:13) ~[create-1.18.2-0.5.0.i.jar%2347!/:0.5.0.i]
	at com.simibubi.create.foundation.block.connected.CTSpriteShifter.getCT(CTSpriteShifter.java:20) ~[create-1.18.2-0.5.0.i.jar%2347!/:0.5.0.i]
	at com.github.talrey.createdeco.connected.SpriteShifts.populateMaps(SpriteShifts.java:27) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.connected.SpriteShifts.<clinit>(SpriteShifts.java:19) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.registry.SheetMetal.buildSheetMetalBlock(SheetMetal.java:74) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.registry.SheetMetal.lambda$registerBlocks$31(SheetMetal.java:209) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
	at com.github.talrey.createdeco.registry.SheetMetal.registerBlocks(SheetMetal.java:204) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.Registration.registerBlocks(Registration.java:396) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	at com.github.talrey.createdeco.CreateDecoMod.<init>(CreateDecoMod.java:52) ~[createdeco-1.3.1-1.18.2.jar%2349!/:1.3.1-1.18.2]
	... 14 more
```

**Comment by talrey** [comment]
Oh, that's odd. I suspect auto-import pulled in the wrong class reference somewhere. I'll look into that, thanks.

**Comment by talrey** [comment]
Please let me know if this hasn't been resolved in 1.3.2.
